src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -53,16 +58,16 @@ const TopicPage = ({ pageData }) => {
           <ATIAnalytics atiData={atiAnalytics} />
           <ChartbeatAnalytics title={title} />
           <MetadataContainer
-            title={activePage >= 2 ? pageTitle : title}
-            socialHeadline={title}
+            title={metadataTitle}
+            socialHeadline={metadataTitle}
```

Comment:
This was always just using `title` before, rather than including the pagination after page 1. Should this be `metadataBaseTitle` instead?